src/assets/css/prose.css

```diff
@@ -43,8 +43,7 @@
     p,
     .content,
     h2,
-    h3,
-    li {
```

Comment:
Just checking this is intentional!

Reply:
A fair question! Yes, this is intentional 😄